### DISCUSSION TITLE
What was the name of George Jetson's son?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1150] [Galaxy Trivia] [answered]
**Bonus:** Where do gummy 🐻🐻‍❄️ come from?

**Comment by tomerf** [comment, marked as answer]
George Jetson's son's name is Elroy.

Gummibären (gummy bears) are a German invention. They were first created by Hans Riegel all the way back in 1922. Hans Riegel was the founder of the German sweet company Haribo and came from the city of Bonn in Germany.

**Comment by thuongtruong1009** [comment]
George Jetson's son is named "Elroy Jetson." Elroy is one of the main characters in the animated television series "The Jetsons," which aired from 1962 to 1963 and depicted a futuristic family living in the year 2062.

Gummy bears, also known as gummy bears or gummy candy, are typically made from a mixture of gelatin, sweeteners, flavorings, and food colorings. They are manufactured in candy factories and come in various shapes and flavors. Gummy bears are a popular type of gummy candy and are widely available in stores.
